src/test/java/com/ankoma88/datastructures/Performance/domain/performance/AnalyzerTests.java

```diff
@@ -0,0 +1,48 @@
+package com.ankoma88.datastructures.performance.domain.performance;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+
+public class AnalyzerTests {
+
+    private TestAnalyzer testAnalyzer;
+
+    @BeforeEach
+    public void setup() {
+        testAnalyzer = new TestAnalyzer();
+    }
+
+    @Test
+    public void testDoPerformanceAnalysis() {
+        long result = testAnalyzer.doPerformanceAnalysis();
+        assert (result >= 10);
```

Comment:
Use static methods from the Assertions class, such as assertTrue, assertEquals, etc., for more readable and informative tests.